tools/auction-deploy/src/auction_deploy/core.py

```diff
@@ -152,6 +183,20 @@ def get_deployed_auction_contracts(
     return deployed_auction_contracts
 
 
+def get_bid_token_address(web3, auction_address: str):
+    compiled_contracts = load_contracts_json(__name__)
+    auction_abi = compiled_contracts["TokenValidatorAuction"]["abi"]
+    auction = web3.eth.contract(address=auction_address, abi=auction_abi)
+    try:
+        return auction.functions.bidToken().call()
+    except BadFunctionCallOutput:
+        # Thrown by web3 when function does not exist on contract
+        return None
+    except TransactionFailed:
```

Comment:
How did you test the BadFunctionCallOutput if this is raised by the tester?
I dont like that we here handle the tester specific, I would hope we can somehow remove this.

Reply:
I tested the BadFunctionCallOutput by deploying an auction on goerli and calling status on it